go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -1141,21 +1142,23 @@ func flowsForDefaultBridge(bridge *bridgeConfiguration, extraIPs []net.IP) ([]st
 				defaultOpenFlowCookie, ofPortHost, types.V4OVNMasqueradeIP, OVNMasqCTZone))
 	}
 	if config.IPv6Mode {
-		// table0, Geneve packets coming from external. Skip conntrack and go directly to host
-		// if dest mac is the shared mac send directly to host.
-		dftFlows = append(dftFlows,
-			fmt.Sprintf("cookie=%s, priority=205, in_port=%s, dl_dst=%s, udp6, udp_dst=%d, "+
-				"actions=output:%s", defaultOpenFlowCookie, ofPortPhys, bridgeMacAddress, config.Default.EncapPort,
-				ofPortHost))
-		// perform NORMAL action otherwise.
-		dftFlows = append(dftFlows,
-			fmt.Sprintf("cookie=%s, priority=200, in_port=%s, udp6, udp_dst=%d, "+
-				"actions=NORMAL", defaultOpenFlowCookie, ofPortPhys, config.Default.EncapPort))
+		if ofPortPhys != "" {
```

Comment:
Have you had the chance to check if by removing some of the flows, traffic might hit other unintended flows?
Meaning we might need a flow to drop traffic that should not be going anywhere.

Reply:
I don't find a case that traffic hits other unintended flows. The removed flows are either from the uplink port or in table-1.